pkgs/cosmic-ext-applet-caffeine/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  libcosmicAppHook,
+  just,
+  pkg-config,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "cosmic-ext-applet-caffeine";
+  version = "0-unstable-2025-02-05";
+
+  src = fetchFromGitHub {
+    owner = "tropicbliss";
+    repo = "cosmic-ext-applet-caffeine";
+    rev = "9fc49bb62377f06cde53b10d46cc33efad9f3a10";
+    hash = "sha256-hWa1pvBNE2K4p/BqneXrC9wxSvRmrDlCp8gDgjn4Kng=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-xTJwVus28p7rNbfYRANo1UYkXDvwOc4ozuu+kPM3GDI=";
+
+  nativeBuildInputs = [
+    libcosmicAppHook
+    just
+    pkg-config
```

Comment:
```suggestion
```

There are no `buildInputs`, so I don't think `pkg-config` needs to be here

Reply:
Done!